docs/source/user_guide/code-snippets.md

```diff
@@ -122,6 +122,24 @@ elyra-metadata install code-snippets \
 
 Refer to the [Code snippet properties](#code-snippet-properties) section for an explanation of the parameters. Note that you must specify the `--name` parameter and the `--replace` parameter. 
 
+#### Exporting code snippets
+
+To export code snippets:
+
+```bash
+elyra-metadata export code-snippets \
+	--schema-name="code-snippet" \
+	--directory="/tmp/foo"
+```
+
+The above example will export all code snippets in the "code-snippet" schema to the "/tmp/foo/code-snippets" directory.
+
+Note that you must specify the `--directory` option. If the `--schema-name` option is not specified, code snippets across all schemas will be exported. 
```

Comment:
```suggestion
Note that you must specify the `--directory` option. If the `--schema_name` option is not specified, code snippets across all schemas will be exported. 
```

Reply:
Looks like I've made this mistake in multiple places - I'll do a find and replace so I'll get them all.

Reply:
Since there's only one schemaspace in the `code-snippets` namespace, we should not display the `--schema_name="code-snippets"`  option in the example and the explanatory text below.

Reply:
Removed all mentions of `--schema-name` parameter from the documentation as discussed via chat.